monetio/sat/_omps_nadir_mm.py

```diff
@@ -13,30 +13,35 @@ def read_OMPS_nm(files):
     import xarray as xr
 
     count = 0
-    print(files)
     # Check if files are url
     if "https" in files[0]:
         filelist = sorted(files)
         for filename in filelist:
             data = extract_OMPS_nm_opendap(filename)
-            # print(data)
             if count == 0:
                 data_array = data
                 count += 1
             else:
                 data_array = xr.concat([data_array, data], "x")
     else:
-        filelist = sorted(glob(files, recursive=False))
-
+        if isinstance(files,str):
+            filelist = sorted(glob(files, recursive=False))
+        else: 
+            filelist = sorted(files) # assume list
         for filename in filelist:
-            data = extract_OMPS_nm(filename)
-
-            if count == 0:
-                data_array = data
-                count += 1
-            else:
-                data_array = xr.concat([data_array, data], "x")
-
+            try:
+                data = extract_OMPS_nm(filename)
+                if count == 0:
+                    data_array = data
+                    count += 1
+                else:
+                    data_array = xr.concat([data_array, data], "x")
+            except KeyError as e:
+                print(f'KeyError occured for: {filename}')
+                print(e)
+            except ValueError as e:
+                print(f'ValueError occured for: {filename}')
+                print(e)
```

Comment:
```suggestion
            except (KeyError, ValueError) as e:
                print(f"warning: skipping {filename}. {type(e).__name__} occured: {e}")
```
I thought I had this in my review but it seems to have disappeared

Reply:
Yeah, I went to test a clean install and got error messages for reading in my downloaded test file set that I would have already dealt with.

The KeyError exception exists because I've run into files that contain no data and are essentially just a filename. The output message is "Unable to synchronously open object (object 'GeolocationData' doesn't exist)".   
I also came across an OMPS observation file where the y-dimension (satellite cross track) was very wrong for some reason and should be skipped.